CMakeLists.txt

```diff
@@ -478,6 +478,66 @@ if (OPENSSL_FOUND AND CUDAQ_ENABLE_REST)
   add_subdirectory(tpls/Crow)
 endif()
 
+# AWS SDK for C++
+set(AUTORUN_UNIT_TESTS OFF CACHE BOOL "" FORCE)
+set(AWS_SDK_WARNINGS_ARE_ERRORS OFF CACHE BOOL "" FORCE)
+set(AWS_USER_AGENT_CUSTOMIZATION "CUDA-Q/${CUDA_QUANTUM_VERSION}" CACHE STRING "" FORCE)
+set(BUILD_ONLY "braket;s3-crt;sts" CACHE STRING "" FORCE)
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
+set(CMAKE_COMPILE_WARNING_AS_ERROR OFF CACHE BOOL "" FORCE)
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
```

Comment:
Please move the definition of crypto_LIBRARY and crypto_INCLUDE_DIR to the rest of the similar variables (line 164), and match the pattern of only defining the variables if they are not already defined yet, and if the values set actually exist.

Reply:
Fixed in 63d84bbbe956e72a2737b6cce1123ef301823002